tests/torch_datasets/test_pvnet_uk.py

```diff
@@ -85,54 +89,62 @@ def test_compute():
 
 
 def test_pvnet_uk_regional_dataset(pvnet_config_filename):
```

Comment:
I would vote to leave this test how it is, and make a new one with dataloader, perhaps `test_pvnet_uk_regional_dataloader`

Reply:
Just to reopen this, could `test_pvnet_uk_regional_dataset` this be left how it before this PR. 

And in this PR, add a new test called `test_pvnet_uk_regional_dataset_dataloader` that tests the dataloader stuff